File: backend/src/main/java/ch/zuehlke/hatch/sailingserver/SailingServerApplication.java
package ch.zuehlke.hatch.sailingserver;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.reactive.config.EnableWebFlux;

@SpringBootApplication
public class SailingServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(SailingServerApplication.class, args);
    }

}

File: backend/src/main/kotlin/ch/zuehlke/hatch/sailingserver/SailingServerApplication.kt
package ch.zuehlke.hatch.sailingserver

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableConfigurationProperties
class SailingServerApplication

fun main(args: Array<String>) {
    runApplication<SailingServerApplication>(*args)
}

